### DISCUSSION TITLE
validating-nested-dataannotation-options-recursively-with-minivalidation/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#380] [Announcements]
# validating-nested-dataannotation-options-recursively-with-minivalidation/

In this post

https://andrewlock.net/validating-nested-dataannotation-options-recursively-with-minivalidation/

**Comment by KAMINM** [comment]
please verify   https://github.com/andrewlock/blog-examples/tree/master/ioptions-validation-minivalidation

**Comment by andreasnilsen** [comment]
Looks like validation of nested types will be supported natively by .NET 8 https://github.com/dotnet/runtime/pull/90275
